cli/src/main.rs

```diff
@@ -109,62 +109,63 @@ fn run<I: IntoIterator<Item = T>, T: Into<OsString> + Clone>(args: I) -> Result<
         )
     );
 
-    app = app.subcommand(
-        SubCommand::with_name("cert")
-            .about("Generates certificates that can be used for development")
-            .setting(AppSettings::SubcommandRequiredElseHelp)
-            .subcommand(
-                SubCommand::with_name("generate")
-                    .long_about(
-                        "Generates test certificates and keys for running splinterd with \
+    app = app
+        .subcommand(
+            SubCommand::with_name("cert")
```

Comment:
This reformatting is odd - did that happen automatically?

Reply:
Yeah that's what `cargo fmt` gave me.